src/Microsoft.AspNet.Mvc.Core/ModelBinding/Validation/DefaultComplexObjectValidationStrategy.cs

```diff
@@ -12,6 +13,8 @@ namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
     /// </summary>
     public class DefaultComplexObjectValidationStrategy : IValidationStrategy
     {
+        private static readonly bool IsMono = Type.GetType("Mono.Runtime") != null;
```

Comment:
static readonly fields are already lazy. They are evaluated on first use of the class.